AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts

```diff
@@ -174,8 +229,30 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
                 this.hearing.participants[indexOfJudge].display_name = this.judge.display_name;
             }
         }
-        const text = SanitizeInputText(this.judgeDisplayName.value);
-        this.judgeDisplayName.setValue(text);
+        const text = SanitizeInputText(this.judgeDisplayNameFld.value);
+        this.judgeDisplayNameFld.setValue(text);
+    }
+
+    changeEmail() {
+        let judgeEmail = this.judgeEmail;
```

Comment:
you could do
```ts
const judgeEmail = this.judgeEmail || this.selectedJudgeEmail;
```

Reply:
that'll just use the left one if it's truthy, or the right one if not

Reply:
selectedJudgeEmail is the Court account assigned email, so I probably need to change to the name to make that more apparent. It is used to store the email if the VHO removes the email from the email input.

Reply:
renamed variable to courtAccountJudgeEmail